lib/velum/secrets.rb

```diff
@@ -0,0 +1,21 @@
+require "securerandom"
+
+module Velum
+  # If there is a rails key base stored on disk, return the stored one.
+  # Otherwise, create a new key base from cryptographic random pool and store it.
+  class Secrets
+    def self.read_or_create_secret_key_base(pathname)
+      pathname.dirname.mkpath
+      key_base = nil
+      if pathname.exist?
+        content = pathname.read
```

Comment:
Maybe add a `.strip` just in case there is some whitespace in the file? Just to make sure the next line works.